book/doctrine.rst

```diff
@@ -722,6 +722,33 @@ instead of querying for rows on a table (e.g. ``product``).
 When querying in Doctrine, you have two options: writing pure Doctrine queries
 or using Doctrine's Query Builder.
 
+Querying for Objects with DQL
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Instead of using the ``QueryBuilder``, you can alternatively write the queries
```

Comment:
Maybe we should change this sentence. The reader doesn't know anything about the query builder yet when reading this paragraph.